### ISSUE TITLE
Regression: From Linux Mint 21 to Linux Mint 21.1 the caps lock icon doesn't show when caps lock is activated

### ISSUE THREAD

**Issue by KMetalmind** [linuxmint/slick-greeter#211]
```
 * slick-greeter version - Latest version
 * Distribution - Mint 21.1 Vera (didn't happen on 20.3 Una or 21 Vanessa)
 * 64 bit
 ```

**Issue**

The caps lock icon doesn't show when caps lock is activated.

**Steps to reproduce**

When typing on the password field, press the caps lock key on the keyboard and no icon or text will show up. Until Mint 21 Vanessa, if you activated caps lock, an icon with an arrow would show up on the password field, so you could know caps lock is activated. 

**Expected behaviour**

When caps lock is activated, there should be something indicating that it is activated. Otherwise, as the password field just has a bunch of "*", you can't know if it's activated or not if the keyboard hasn't got a led showing that info. It could be an icon (the same or other) or a text. Any kind of info.

**Comment by clefebvre** [comment]
I can't reproduce the issue. That said, the slick-greeter theme changed in 21.1 from Mint-X to Mint-Y-Aqua and this results in the warning icon being less visible.

**Comment by clefebvre** [comment]
Fixed in mint-themes 2.0.9
